cunumeric/random/random.py

```diff
@@ -43,6 +50,50 @@ def beta(
     size: Union[NdShapeLike, None] = None,
     dtype: npt.DTypeLike = np.float64,
 ) -> ndarray:
+    """
+    beta(a, b, size=None)
+
+    Draw samples from a Beta distribution.
+
+    The Beta distribution is a special case of the Dirichlet distribution,
+    and is related to the Gamma distribution.  It has the probability
+    distribution function
+
+    .. math:: f(x; a,b) = \\frac{1}{B(\\alpha, \\beta)} x^{\\alpha - 1}
+                                                     (1 - x)^{\\beta - 1},
+
+    where the normalization, B, is the beta function,
+
+    .. math:: B(\\alpha, \\beta) = \\int_0^1 t^{\\alpha - 1}
+                                 (1 - t)^{\\beta - 1} dt.
+
+    It is often seen in Bayesian inference and order statistics.
+
+    Parameters
+    ----------
+    a : float or array_like of floats
```

Comment:
The documentation claims this function can take `array_like`s of floats, but the type signature (and AFAICT the implementation) only accept scalar values. Also the case for other functions in this file.

Reply:
3f68ce4aacfbb28323d4ef611e7badaccfed9ac4